google/cloud/storagecontrol/v2/.repo-metadata.json

```diff
@@ -8,7 +8,7 @@
     "library_type": "GAPIC_AUTO",
     "name_pretty": "Cloud Storage API",
     "product_documentation": "https://cloud.google.com/storage/docs/overview",
-    "release_level": "preview",
+    "release_level": "stable",
```

Comment:
Sorry, I forgot to add this comment to the review....

Based on #13513 it looks like we can remove the `omit_repo_metadata` field from the generator config now.

https://github.com/googleapis/google-cloud-cpp/blob/9d16ff3af7e44b0beaf31bc21cb2b77eb0ab5e7c/generator/generator_config.textproto#L3354